manifests/a/AppManager/wahke/1.0/AppManager.wahke.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.10.0.schema.json
+
+PackageIdentifier: AppManager.wahke

```

```suggestion
PackageIdentifier: wahke.AppManager
```
Please update file name and path to match this change.